tech-debt.md

```diff
@@ -2,5 +2,88 @@
 
-We don't currently have a strategy for technical debt, at least not documented. This document aims to do that.
+Tech Debt is an inevitable part of programming, a valuable tool even. Do not shy away from it. Embrace it! Subdue it! Make it your servant. Whatever you do, *do not just ignore it* and hope it goes away.
 
+*Note:* This document is aspirational, it doesn't necessarily reflect the current culture of technical debt at Holiday Extras. It does however reflect where we would like to be, and is a work in progress. If you have ideas about how to better manage technical debt please tell people. Good places to do this are the #developers, #guid-front-end and #guild-back-end slack channels or with a PR for this page.
+
+## The Seven Commandments of Tech Debt
+
+### 1. Be Courageous! Push Back When Appropriate.
+
+This commandment comes from our general company values. All our values are relevant however this one particularly so.
+
+*Be courageous* and push back against pressure to ship more and more technical debt without considerations for the long term implications. Be respectful of other stakeholders but make sure that you speak up when you think the issue of technical debt is not being addressed correctly.
+
+### 2. Anticipate Tech Debt From the Beginning
+
+Tech Debt to some extent is part and parcel of every piece of code that you write. Before you lay down a single line of code, you should be thinking about tech debt. Perhaps even as soon as you are aware of upcoming work. It is never too early to be thinking about:
+
+* existing debt that might slow you down
+* existing debt that it makes sense to pay off
+* potential new debt that it makes sense to accrue
+* how you might be able to avoid adding more tech debt
+
+### 3. Let's Talk About Tech Debt
+
+Once you have some picture of the above you should be constantly communicating your thoughts on tech debt to others:
+
+* your pod lead
+  * tech debt impacts cost and cost impact priorities
+  * any decision to accrue tech debt *has to* accept the long term cost
+* developers/software engineers
+  * you may have insights into tech debt that will impact their work (and they yours)
+  * you may be able to help them avoid tech debt in the first place (and they you)
+* software architects
+  * they have some of the best knowledge on existing tech debt
+  * they will be able to advise how to avoid tech debt in the first place
+  * they can help make your case to others that a certain piece of tech debt needs to be addressed
+  * you can help keep them up to date with the general picture of tech debt
+
+#### Where to do this?
+
+The existing communication channels are perfect for talking about tech debt. We just need to make sure we are using them.
+
+* *Stand ups* can be a great place to flag any technical debt you are facing or expecting to face in the immediate term.
```

Comment:
We could also suggest Scrum of Scrums, but I guess that is just a really long standup